platform-sdk/swirlds-unit-tests/core/swirlds-platform-test/src/test/java/com/swirlds/platform/test/consensus/ConsensusTests.java

```diff
@@ -128,7 +175,7 @@ void subQuorumPartitionTests(final ConsensusTestParams params) {
     void cliqueTests(final ConsensusTestParams params) {
         ConsensusTestRunner.create()
                 .setTest(ConsensusTestDefinitions::cliqueTests)
-                .setParams(params)
+                .setParams(modifyParams(params))
                 .setIterations(NUM_ITER)
                 .run();
```

Comment:
Any reason you're not asserting marker file presence on this test?

Reply:
oversight, thanks for the catch.